config/boards/odroidn2.conf

```diff
@@ -1,12 +1,14 @@
 # Amlogic S922X hexa core 2GB/4GB RAM SoC 1.8-2.4Ghz eMMC GBE USB3 SPI RTC
 BOARD_NAME="Odroid N2"
 BOARDFAMILY="meson-g12b"
-BOOTCONFIG="odroidn2_config"
-CPUMIN=1000000
-CPUMAX=2400000
+BOOTCONFIG="odroid-n2_defconfig"
 MODULES_LEGACY="media_clock firmware #decoder_common #stream_input #amvdec_mh264 #amvdec_h264 #amvdec_h264mvc #amvdec_h265 #amvdec_mmjpeg #amvdec_mpeg12 #amvdec_mmpeg4 #amvdec_mpeg4 #amvdec_vc1 #amvdec_vp9"
```

Comment:
Removing whole line "MODULES_LEGACY"

Reply:
I kept since we can keep legacy after all.

Reply:
Nah, I finally see what you mean, MODULES_LEGACY is not used anywhere and is thus cruft and should be removed. Thanks